app/logic/courseManagement.py

```diff
@@ -45,6 +47,56 @@ def approvedCourses(termId):
 
     return approvedCourses
 
+
+def importedCourses(termId):
+    '''
+    Queries the database to get all the necessary information for 
+    imported courses.
+    '''
+    importedCourses = (Course.select(Course, Term, CourseStatus, fn.GROUP_CONCAT(" " ,User.firstName, " ", User.lastName).alias('instructors'))
+                        .join(CourseInstructor, JOIN.LEFT_OUTER)
+                        .join(User, JOIN.LEFT_OUTER).switch(Course)
+                        .join(CourseStatus).switch(Course)
+                        .join(Term)
+                        .where(Term.id == termId, Course.status == CourseStatus.IMPORTED)
+                        .group_by(Course, Term, CourseStatus))
+
+    return importedCourses
+
+
+def editImportedCourses(courseData, attachments=None):
+    """
+        This function will take in courseData for the SLC proposal page and a dictionary
+        of instructors assigned to the imported course after that one is edited 
+        and update the information in the db.
+    """
+
+    with mainDB.atomic() as transaction:
+        try:
+            course = Course.get_by_id(courseData["courseId"])
+            (Course.update(
```

Comment:
Channeling my inner Anderson to tell you to make the indentations all on the same line for this query.

Reply:
Fixed.
Don't let him see. It would be too much of a praise.